src/odb/src/db/dbJournal.cpp

```diff
@@ -392,12 +392,20 @@ void dbJournal::redo_createObject()
     case dbModITermObj: {
       std::string name;
       uint obj_id;
+      uint modbterm_obj_id;
       uint parent_obj_id;
       _log.pop(name);
       _log.pop(obj_id);
+      _log.pop(modbterm_obj_id);
       _log.pop(parent_obj_id);
       dbModInst* parent_mod_inst = dbModInst::getModInst(_block, parent_obj_id);
       dbModITerm* mod_iterm = dbModITerm::create(parent_mod_inst, name.c_str());
+      dbModBTerm* mod_bterm = nullptr;
+      if (modbterm_obj_id != 0) {
+        mod_bterm = dbModBTerm::getModBTerm(_block, modbterm_obj_id);
+      }
+      mod_iterm->setChildModBTerm(mod_bterm);
+      mod_bterm->setParentModITerm(mod_iterm);
```

Comment:
Why not pass the mod_bterm to dbModITerm::create now that it takes it as an argument?